src/ViewModels/CkanGroupViewModel.js

```diff
@@ -339,23 +374,31 @@ function populateGroupFromResults(viewModel, json) {
             // Extract the layer name from the WMS URL.
             var uri = new URI(wmsUrl);
```

Comment:
It's confusing that this variable is called `wmsUrl` when it's no longer necessarily a WMS URL.